doc/tutorial/argument_parser/disable_version_check.cpp

```diff
@@ -1,9 +1,7 @@
 #include <seqan3/argument_parser/all.hpp>
 
-using namespace seqan3;
-
 int main(int argc, char ** argv)
 {
-    argument_parser myparser{"Game-of-Parsing", argc, argv, false};
+    seqan3::argument_parser myparser{"Game-of-Parsing", argc, argv, false};
     // disable version checks permanently --------------------^
```

Comment:
add some more `----` such that the small "arrow" points to the `false` parameter :nail_care: